addons/website/static/src/snippets/s_dynamic_snippet/000.xml

```diff
@@ -30,5 +30,9 @@
                 </t>
             </div>
         </t>
+        <!-- Load More button -->
+        <div class="text-center mt-3">
+            <button class="btn btn-primary s_dynamic_snippet_load_more d-none">Load More</button>
```

Comment:
No load more button even though there are 32 records and I have set Fetched Elements 8
<img width="1913" height="268" alt="image" src="https://github.com/user-attachments/assets/b1608e56-9196-449f-9232-a6ffaf17958e" />
<img width="1505" height="552" alt="image" src="https://github.com/user-attachments/assets/cd8d5172-e2bd-44e7-8ad5-9470751da6af" />